src/catalog/src/remote/manager.rs

```diff
@@ -386,7 +386,11 @@ impl CatalogManager for RemoteCatalogManager {
             "Initialized catalogs: {:?}",
             catalogs.keys().cloned().collect::<Vec<_>>()
         );
-        self.catalogs.store(Arc::new(catalogs));
+
+        catalogs.into_iter().for_each(|(k, v)| {
+            self.catalogs.insert(k, v);
+        });
```

Comment:
Can we use `extend` here?

Reply:
Seems not,  extend need `&mut self`
`extend()` source code is as follows:
```rust
impl<K: Eq + Hash, V, S: BuildHasher + Clone> Extend<(K, V)> for DashMap<K, V, S> {
    fn extend<I: IntoIterator<Item = (K, V)>>(&mut self, intoiter: I) {
        for pair in intoiter.into_iter() {
            self.insert(pair.0, pair.1);
        }
    }
}

```